src/Global/atoms/NavigationItem.tsx

```diff
@@ -94,8 +96,8 @@ const SubmenuContainer = styled.div`
 
 `;
 
-const ContextContainer = styled.div<{ open: boolean, maxHeight: number, loading: 'true'|'false' }>`
-  min-height: 70px;
+const ContextContainer = styled.div<{ open: boolean, maxHeight: number, minHeight?: number, loading: 'true'|'false' }>`
+  min-height: ${({minHeight}) => minHeight ? `${minHeight}px` : `70px`};
```

Comment:
Minor: I wonder, Is it intentional that the units on this min;height are only in `px` would it be more flexible to allow any valid unit?